firstvoices/backend/search/query_builder.py

```diff
@@ -33,4 +35,14 @@ def get_search_query(q=None, site_slug=None):
     if site_slug:
         search_query = search_query.query(get_site_filter_query(site_slug))
 
+    # Adding type filters
+    # If only one of the "words" or "phrases" is present, we need to filter out the other one
+    # no action required if both are present
+    if "words" in types and "phrases" not in types:
+        search_query = search_query.query(
+            ~Q("match", type=TypeOfDictionaryEntry.PHRASE)
+        )
+    elif "phrases" in types and "words" not in types:
+        search_query = search_query.query(~Q("match", type=TypeOfDictionaryEntry.WORD))
```

Comment:
Is there a reason to do the filtering as "is not a phrase" instead of "is a word"?

Reply:
Yes, so adding in "is a word" instead of "is not a phrase" will result in more modification of queries while this will require minimal modification even when we add songs and stories due to each document type having their own index